lib/ui/app/list_scaffold.dart

```diff
@@ -68,6 +69,7 @@ class ListScaffold extends StatelessWidget {
         builder: (context) => GestureDetector(
           onLongPress: onHamburgerLongPress,
           child: IconButton(
+            key: Key(kDrawerKey),
```

Comment:
Not a big deal but it may be better if we can use a tooltip instead of the key to find the element in a test